tests/tests/blob.rs

```diff
@@ -151,3 +168,19 @@ async fn blob__accessing_nonexitent_blob_panics_vm() {
     // Then
     assert!(matches!(tx_status, TransactionStatus::Failure { .. }));
 }
+
+#[tokio::test]
+async fn blob__can_be_queried_if_uploaded() {
+    // Given
+    let mut ctx = TestContext::new().await;
+
+    // When
+    let bytecode: Vec<u8> = [op::ret(RegId::ONE)].into_iter().collect();
+    let (status, blob_id) = ctx.new_blob(bytecode.clone()).await.unwrap();
+    assert!(matches!(status, TransactionStatus::Success { .. }));
+
+    // Then
+    let result = ctx.query_blob(blob_id).await;
+    assert!(result.contains(&format!(r#""id":"0x{}"#, blob_id)));
+    assert!(result.contains(&format!(r#""bytecode":"0x{}""#, hex::encode(bytecode))));
+}
```

Comment:
[nit]: the SUT of this test would be the query.

```suggestion
#[tokio::test]
async fn blob__can_be_queried_if_uploaded() {
    // Given
    let mut ctx = TestContext::new().await;
    let bytecode: Vec<u8> = [op::ret(RegId::ONE)].into_iter().collect();
    let (status, blob_id) = ctx.new_blob(bytecode.clone()).await.unwrap();
    assert!(matches!(status, TransactionStatus::Success { .. }));

    // When
    let result = ctx.query_blob(blob_id).await;
    
    // Then
    assert!(result.contains(&format!(r#""id":"0x{}"#, blob_id)));
    assert!(result.contains(&format!(r#""bytecode":"0x{}""#, hex::encode(bytecode))));
}
```

Reply:
Makes sense, thanks for suggesting!